src/main/java/com/jfrog/ide/idea/ui/export/ExportCsvViolatedLicenses.java

```diff
@@ -0,0 +1,34 @@
+package com.jfrog.ide.idea.ui.export;
+
+import com.intellij.openapi.project.Project;
+import com.jfrog.ide.common.exporter.Exporter;
+import com.jfrog.ide.idea.log.Logger;
+import org.apache.commons.lang3.exception.ExceptionUtils;
+
+import java.awt.event.ActionEvent;
+
+/**
+ * Represents export violated licenses to CSV menu button action.
+ *
+ * @author yahavi
+ **/
+class ExportCsvViolatedLicenses extends ExportCsv {
+    private static final String DEFAULT_VIOLATED_LICENSES_FILE_NAME = "violated-licenses.csv";
+
+    public ExportCsvViolatedLicenses(Project project) {
+        super(project, "Violated Licenses");
+    }
+
+    @Override
+    public void actionPerformed(ActionEvent e) {
+        try {
+            Exporter csvExporter = createCsvExporter(project);
+            if (csvExporter == null) {
+                return;
+            }
+            exportIfNeeded(project, csvExporter.generateViolatedLicensesReport(), DEFAULT_VIOLATED_LICENSES_FILE_NAME);
```

Comment:
The above code - 
```
            if (csvExporter == null) {
                return;
            }
            exportIfNeeded(project, csvExporter.generateViolatedLicensesReport(), DEFAULT_VIOLATED_LICENSES_FILE_NAME);
```
can be shortened to - 
```
            if (csvExporter != null) {
                exportIfNeeded(project, csvExporter.generateViolatedLicensesReport(), DEFAULT_VIOLATED_LICENSES_FILE_NAME);
            }
```
The same goes for the code in the ```ExportCsvVulnerabilities``` class.